Framework/Muon/src/CalMuonDetectorPhases.cpp

```diff
@@ -398,7 +398,7 @@ void CalMuonDetectorPhases::getGroupingFromInstrument(
   const auto instrument = ws->getInstrument();
   auto loader = Kernel::make_unique<API::GroupingLoader>(instrument);
 
-  if (instrument->getName() == "MUSR") {
+  if (instrument->getName() == "MUSR" || instrument->getName() == "CHRONUS") {
```

Comment:
As a long term point, would we be able to move toward not checking instrument names and maybe use instrument parameters, like indirect do for things like this?
Cc @AnthonyLim23 as it's a more long-term thing.

Reply:
yes it would make sense